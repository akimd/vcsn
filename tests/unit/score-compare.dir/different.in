 0.24s: a.is_proper()        # a = "", 200000x
 0.18s: b.format("text")     # b = [abc] -> B, 90000x
