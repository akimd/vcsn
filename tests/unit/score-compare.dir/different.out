[32m[1m 0.12[0m [31m[1m 0.24[0m a.is_proper()        # a = "", 200000x
[31m[1m 2.43[0m [32m[1m 1.80[0m b.format("text")     # b = [abc] -> B, 900000x
